src/_designs/styles/mixins.scss

```diff
@@ -0,0 +1,43 @@
+@import "variables";
+
+@mixin xs {
```

Comment:
- I have removed the mixins and use `@media` instead.
- also I'm using `display:flex` instead of `display:table` and its child styles to improve readability
- I'm now also referencing the relevant import files instead of general index